scripts/compile_all_examples.py

```diff
@@ -105,12 +108,24 @@ class CompilationResult:
 
 def get_program_size(path: Path) -> int:
     try:
-        program = algokit_utils.Program(path.read_text("utf-8"), ALGOD_CLIENT)
+        program = algokit_utils.Program(replace_templates(path.read_text("utf-8")), ALGOD_CLIENT)
         return len(program.raw_binary)
     except Exception as e:
         raise Exception(f"Error compiling teal application: {path}") from e
 
 
+def replace_templates(
```

Comment:
Have to replace templates with something valid before compiling with algod or it goes 💥